reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -157,8 +156,7 @@ def submit(self, spec, parameters, state, metadata):
 
     def result(self, resultproxy):
         """Retrieve the result of a pactivity run by RJC."""
-        resultproxy.pars = contextualize_parameters(resultproxy.pars,
-                                                    resultproxy.state)
+        # resultproxy.pars = contextualize_parameters(resultproxy.pars, resultproxy.state)
```

Comment:
dito, this will fail because `resultproxy.pars` is used in the line below

Reply:
Is there a documentation for the new package? (sorry if there's one, but could not find it at assumed Occam's razor for simplicity)

Reply:
Have traced the `contextualize_parameters` function and it is now named `finalize_inputs`. Could you confirm this is the one?